game-core/src/main/java/games/strategy/triplea/ai/pro/util/ProTerritoryValueUtils.java

```diff
@@ -92,12 +84,12 @@ public static double findTerritoryAttackValue(final PlayerId player, final Terri
    * Returns the value of each sea territory in {@link ProData#getData()}.
    */
   public static Map<Territory, Double> findSeaTerritoryValues(final PlayerId player,
-      final List<Territory> territoriesThatCantBeHeld) {
+      final List<Territory> territoriesThatCantBeHeld, final List<Territory> territoriesToCheck) {
```

Comment:
Always pass in a list of territories to check to avoid checking every territory on the map.